client/components/settings/settingBody.jade

```diff
@@ -0,0 +1,72 @@
+template(name="setting")
```

Comment:
Thanks for comment, I tried follow the naming convention of the project Wekan itself, where I saw there is not this kinda rule about the name of a template should be plural, take components/boards/boardBody.jade as example.